phpstan-unsupported-baseline.neon

```diff
@@ -1,26 +1,17 @@
 parameters:
 	ignoreErrors:
 		-
-			message: "#^Parameter \\#1 \\$callable of class loophp\\\\collection\\\\Collection constructor expects callable\\(\\.\\.\\.mixed\\)\\: iterable\\<int, string\\>, Closure\\(string, string\\)\\: loophp\\\\collection\\\\Iterator\\\\StringIterator\\<mixed, string\\> given\\.$#"
+			message: "#^Method loophp\\\\collection\\\\Collection\\:\\:squash\\(\\) should return loophp\\\\collection\\\\Contract\\\\Collection\\<TKey, T\\> but returns loophp\\\\collection\\\\Contract\\\\Collection\\<int, array\\(TKey, T\\)\\>\\.$#"
```

Comment:
this error is new and my bet is it's due to the type updates in `unpack`

Reply:
Yes, this is a big mystery for me. I don't know where this is coming from yet. I added it in that file, but I'm planning to remove it. I might need some help on this thing if you have time.

Reply:
This has been fixed now that I reverted the change in `Unpack`.